native/src/modules/endpoint/DatabaseConnector.js

```diff
@@ -466,6 +479,7 @@ class DatabaseConnector {
         pushNotificationsEnabled: jsonObject.pushNotificationsEnabled,
         tunewsEnabled: jsonObject.tunewsEnabled,
         offersEnabled: jsonObject.extras_enabled,
+        poisEnabled: jsonObject.pois_enabled || false,
```

Comment:
pois_enabled should never be undefined, should it.

Reply:
Hmm I admit this should be redundant most of the time. However, after upgrading the app without a fetch/store of the cities the old cities won't have this flag, so I am setting it to false otherwise here. Kind of does not matter because undefined is falsy as well. I don't mind removing it again.

Reply:
then I would prefer if you remove it. Otherwise it would be inconsistent with offersEnabled and tunewsEnabled

Reply:
Will do in #93